hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -361,7 +361,11 @@ public synchronized HoodieActiveTimeline getActiveTimeline() {
    * @return Active instants timeline
    */
   public synchronized HoodieActiveTimeline reloadActiveTimeline() {
-    activeTimeline = new HoodieActiveTimeline(this);
+    if (activeTimeline == null) {
+      activeTimeline = new HoodieActiveTimeline(this);
```

Comment:
revert it back.

Reply:
I modified this, because we should use `activeTimeline.reload()` as much as possible, because user can reload `activeTimeline.reload()`, such as implementing their own incremental reload